prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array ' + languages);
+var firstElement = languages[0];
+console.log('the first element of the array is ' + firstElement);
+var thirdElement = languages[2];
+console.log('the third element of the array is ' + thirdElement);
+var length = languages.length;
+console.log('the length of the array is ' + length);
+var lastElement = languages[5];
```

Comment:
You can get the last element of any array by subtracting 1 from the array's length

```js
var lastElement = languages[length - 1];
```